src/content/pt-BR/tools/chrome-devtools/javascript/reference.md

```diff
@@ -0,0 +1,455 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Descubra novas formas de depuração nesta referência abrangente dos recursos de depuração do Chrome DevTools.
+
+{# wf_updated_on: 2019-01-31 #} {# wf_published_on: 2017-01-04 #} {#
+wf_blink_components: Platform>DevTools #}
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+<style>
+.devtools-list {
+  list-style: none;
+}
+</style>
+
+# Referência de como depurar JavaScript {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+Descubra novas formas de depuração, nesta abrangente referência dos recursos de
+depuração do Chrome DevTools.
+
+Consulte [Primeiros passos com a depuração do JavaScript no Chrome
+DevTools](/web/tools/chrome-devtools/javascript) para aprender o básico de
+depuração.
+
+## Pause o código com breakpoints {: #breakpoints }
+
+Defina um breakpoint para que você possa pausar seu código no meio de sua
+execução.
+
+Consulte [Pausar o código com pontos de interrupção](breakpoints) para aprender
+como definir breakpoints.
+
+## Percorrer o código {: #stepping }
+
+Depois que seu código for pausado, passe por ele, uma linha por vez,
+investigando a estrutura de controle e os valores das propriedades ao longo do
+caminho.
+
+### Passe por uma linha de código {: #step-over }
+
+Quando pausado em uma linha de código que contém uma função que não é relevante
+para o problema que você está depurando, clique em **Step over**![Step
+over](https://github.com/google/WebFundamentals/blob/master/src/content/en/tools/chrome-devtools/javascript/imgs/step-over.png?raw=true)
```

Comment:
Do not link to GitHub URLs, use the original URL as provided.

Reply:
Fixes in #8850